submissions/kazeemErinfolami/javascript/Dom/contact.js

```diff
@@ -0,0 +1,55 @@
+let submit = document.getElementById("formValidation"),
+  fullName = document.getElementById("fullName"),
+  email = document.getElementById("email"),
+  subject = document.getElementById("subject"),
+  message = document.getElementById("message");
+
+//required Error message
+let fullNameRequired = "⚠️Please enter your Name",
+  emailRequired = "⚠️Please enter your Email Address",
+  subjectRequired = "⚠️Please enter a message subject",
+  messageRequired = "⚠️Please enter your Message";
+
+//required validation Error message
+let fullNameError = "no special cha",
+  emailError = "⚠️Please enter a valid email",
+  subjectError = "",
```

Comment:
I think you should use the empty quotes directly instead of creating a variable for it. @kazeemerinfolami